internal/services/order/req_res_body.go

```diff
@@ -1,18 +1,23 @@
 package order
 
+const pending,completed,halfCompleted,failed,cancelled = "PENDING","COMPLETED","HALF_COMPLETED","FAILED","CANCELLED"
```

Comment:
Keep them line separated for better readability.

Reply:
Done.